app/javascript/components/public_health/Import.js

```diff
@@ -286,6 +280,14 @@ class Import extends React.Component {
                         <b>Monitored Zip:</b> {patient.monitored_address_zip}
                       </Col>
                       <Col>
+                        <b>Contact Type:</b> {patient.contact_type || 'Unknown'}
```

Comment:
Don't think this is needed since `patient.contact_type` defaults to `Unknown`.

Reply:
This actually is needed, because if you import a blank contact type, it's not set to "Unknown" yet but it will be so that needs to be shown to the user because contact field cannot be blank
<img width="661" alt="Screen Shot 2021-10-15 at 10 39 38 AM" src="https://user-images.githubusercontent.com/35042815/137505724-fdd779d5-734d-4717-9a7e-a32f1f21808d.png">